addons/binding/org.openhab.binding.allplay/META-INF/MANIFEST.MF

```diff
@@ -14,6 +14,7 @@ Bundle-NativeCode: lib/x86-64/win/alljoyn_java.dll;processor=amd64;osname=win32;
  lib/arm/linux/liballjoyn_java.so;processor=arm;osname=linux,*
 Import-Package: com.google.common.collect,
  org.apache.commons.io,
+ org.eclipse.jdt.annotation;resolution:=optional,
```

Comment:
The new archtype that will be released with the next ESH stable release already includes this as a default, see [here](https://github.com/eclipse/smarthome/blob/master/tools/archetype/binding/src/main/resources/archetype-resources/META-INF/MANIFEST.MF#L9). So please refuse bindings without this dependency.

A static analysis would be nice to have, yes. In the end it is a benefit for the developers as they get a direct warning in there IDE if they pass a `null` value to a `@NonNull` parameter of a method for example.